gmail/src/models/partner.ts

```diff
@@ -197,10 +197,21 @@ export class Partner {
     }
 
     /**
-     * Manually enrich the given partner.
+     * Create and enrich the company of the given partner.
```

Comment:
Isn't it creating if missing ?

Reply:
This can be called only if the partner has no company, otherwise the endpoint will return the error "The partner already has a company related to him"